RecoLocalCalo/HGCalRecProducers/plugins/HGCalCLUEAlgo.cc

```diff
@@ -251,54 +332,109 @@ void HGCalCLUEAlgo::calculateDistanceToHigher(const HGCalLayerTiles& lt, const u
     float maxDelta = std::numeric_limits<float>::max();
     float i_delta = maxDelta;
     int i_nearestHigher = -1;
-
-    // get search box for ith hit
-    // guarantee to cover a range "outlierDeltaFactor_*delta_c"
-    auto range = outlierDeltaFactor_ * delta_c;
-    std::array<int, 4> search_box = lt.searchBox(
-        cellsOnLayer.x[i] - range, cellsOnLayer.x[i] + range, cellsOnLayer.y[i] - range, cellsOnLayer.y[i] + range);
-
-    // loop over all bins in the search box
-    for (int xBin = search_box[0]; xBin < search_box[1] + 1; ++xBin) {
-      for (int yBin = search_box[2]; yBin < search_box[3] + 1; ++yBin) {
-        // get the id of this bin
-        size_t binId = lt.getGlobalBinByBin(xBin, yBin);
-        // get the size of this bin
-        size_t binSize = lt[binId].size();
-
-        // loop over all hits in this bin
-        for (unsigned int j = 0; j < binSize; j++) {
-          unsigned int otherId = lt[binId][j];
-
-          float dist = distance(i, otherId, layerId);
-          bool foundHigher =
-              (cellsOnLayer.rho[otherId] > cellsOnLayer.rho[i]) ||
-              (cellsOnLayer.rho[otherId] == cellsOnLayer.rho[i] && cellsOnLayer.detid[otherId] > cellsOnLayer.detid[i]);
-          // if dist == i_delta, then last comer being the nearest higher
-          if (foundHigher && dist <= i_delta) {
-            // update i_delta
-            i_delta = dist;
-            // update i_nearestHigher
-            i_nearestHigher = otherId;
+    float delta_c_(0);
```

Comment:
drop the trailing underscore and/or rename to ```delta```

Reply:
Done.